WreckEngine/TextEntity.cpp

```diff
@@ -1,6 +1,17 @@
 #include "TextEntity.h"
 
 TextEntity::TextEntity(const std::string _message, const std::string& _font, Text::Justify _vertical, Text::Justify _horizontal, const uint32_t height, const uint32_t width)
-	: Entity(), font(Text::loadWinFont(_font, height, width)), message(_message), vertical(_vertical), horizontal(_horizontal) {}
+    : Entity()
+    , vertical(_vertical)
+    , horizontal(_horizontal)
+    , inst(make_shared<Text::Instance>())
```

Comment:
That would be unreasonable lol. I was wondering whether the idea was something like "I'm using the same text, but the non-baked attributes are different" What was the reasoning behind making it shared?